src/encoded/static/components/genome_browser.js

```diff
@@ -553,6 +553,21 @@ GenomeLegend.propTypes = {
 };
 
 
+/**
+ * Retrieve the target from the given file. Some files have a `targets` array, in which case return
+ *  the first target from this array.
+ * @param {object} file File included in browser
+ * @returns {object} `target` or first `targets` object from file.
+ */
+const getFileTarget = (file) => {
+    let fileTarget = file.targets && file.targets[0];
```

Comment:
I look forward to August when we don't have to do use 20 year-old hacks like this; that make me pause. In August, we can use "Optional chaining" and not worry about IE11 self-destruction.